docs/cryoet_data_portal_docsite_quick_start.md

```diff
@@ -33,13 +34,56 @@ Install the latest `cryoet_data_portal` package via pip:
 pip install -U cryoet-data-portal
 ```
 
-## Python quick start
+## API Functions Overview
 
-Below are 3 examples of common operations you can do with the client. Check out the [examples page](https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_examples.html) for more code snippets.
+The Portal API has functions for searching and downloading data. **Every class** has a `find` and `get_by_id` method for selecting data, and most classes have `download...` methods for downloading the data. Below is a table of the API classes download functions.
 
-### Browse data in the portal
+| **Class**               | **Download Functions**                                                                                 |
+|-------------------------|--------------------------------------------------------------------------------------------------------|
+| Dataset                 | `download_everything`                                                                                  |
+| DatasetAuthor           | None                                                                                                   |
+| DatasetFunding          | None                                                                                                   |
+| Run                     | `download_everything`                                                                                  |
+| TomogramVoxelSpacing    | `download_everything`                                                                                  |
+| Tomogram                | `download_all_annotations`, `download_mrcfile`, `download_omezarr`                                     |
+| TomogramAuthor          | None                                                                                                   |
+| Annotation              | `download`                                                                                            |
+| AnnotationFile          | None                                                                                                   |
+| AnnoatationAuthor       | None                                                                                                   |
+| TiltSeries              | `download_alignment_file`, `download_angle_list`, `download_collection_metadata`, `download_mrcfile`, `download_omezarr` |
```

Comment:
Wondering if this table should contain only those entities that have data files associated? 

i.e. `DatasetAuthor`, `DatasetFunding`, `TomogramAuthor`, `AnnotationAuthor` may be left out?

Also, can we add anchor-links for each class in the table to the API docs?

Reply:
Good point. I like listing all the classes for completeness, but I updated the entries for those 4 classes to:
"Not applicable as this class doesn't contain data files"
For the AnnotationFile class I added, "None, use the Annotation or Tomogram class to download annotations"

Anchor links are added, and I updated all "function" to "method"